packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -193,6 +219,9 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
         this.webCallingService.deregisterWebCallingLine();
       }
 
+      this.taskManager.unregisterIncomingCallEvent();
+      this.taskManager.off(TASK_EVENTS.TASK_INCOMING, this.handleIncomingTask);
```

Comment:
I think this might be better placed in a finally block right?

Reply:
as discussed, station logout can fail due to many reasons. Therefore, we can leave it as it is for now. In the error epic, we'll need to take care of this.